src/integTest/kotlin/io/snyk/plugin/cli/ConsoleCommandRunnerTest.kt

```diff
@@ -238,13 +238,10 @@ class ConsoleCommandRunnerTest : LightPlatformTestCase() {
                 override fun run(indicator: ProgressIndicator) {
                     val output = ConsoleCommandRunner().execute(sleepCommand, getPluginPath(), "", project)
                     assertTrue(
-                        "Should get timeout error, but received:\n$output",
-                        output.startsWith("Execution timeout")
+                        "Should get timeout error, but received:\n$output", output.startsWith("Execution timeout")
                     )
                 }
-            },
-            EmptyProgressIndicator(),
-            null
+            }, EmptyProgressIndicator(), null
```

Comment:
minor: a bit strange formatting here and above... does auto-formatted that way? usually arguments/params placed on separate line if not fit in one line.

Reply:
Yep, autoformatted according to our editorconfig.

Reply:
hmm... we might need to review our `editorconfig` as afaik that is against java/kotlin standard practices...